metrics/time/launch_times.sh

```diff
@@ -104,7 +104,7 @@ run_workload() {
 		kernel_period=$(echo $kernel_last_line | awk '{print $2}' | tr -d "]")
 
 		# And we can then work out how much time it took to get to the kernel
-		to_kernel_period=$(bc <<<"scale=3; $(ns_to_s $workload_period) - $kernel_period")
+		to_kernel_period=$(printf "%.03f" $(bc <<<"scale=3; $(ns_to_s $workload_period) - $kernel_period"))
```

Comment:
here you are cutting off the precision to 3 decimals.
correct me if I'm wrong, but I think the problem @grahamwhaley mentions in #562 is not the precision of the floating point, the problem is the leading 0, so I think you should use `%0f`

Reply:
@devimc , yes I cutting off the precision but is also solving the issue with the leading 0.